tests/testTriggers.go

```diff
@@ -8063,10 +8055,15 @@ func TriggerStorkAppBkpVolResize(contexts *[]*scheduler.Context, recordChan *cha
 					return
 				}
 				log.InfoD("backup successful and volume resize injected during backup successfully, backup name - %v, backup location - %v", backupname, backuplocationname)
+				dashStats, err := stats.GetStorkBackupStats(backupname, currbkNamespace)
+				if err != nil {
+					log.InfoD("Not able to get stats, err: %v", err)
+				}
+				updateLongevityStats(StorkAppBkpVolResize, stats.StorkApplicationBackupEventName, dashStats)
```

Comment:
Is it stats.StorkApplicationBackupEventName? It does not look like it corresponds to TriggerStorkAppBkpVolResize.